.github/workflows/pull.yml

```diff
@@ -66,165 +65,14 @@ jobs:
-
-            https://developers.google.com/speed/pagespeed/insights/?url=https://web.env.reactivetrader.com/pull/${{ github.event.number }}
-
-  web-end-to-end-test:
-    name: Web e2e test - All
-
-    needs: build
-
-    runs-on: ubuntu-20.04
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Set Node version
-        uses: actions/setup-node@v4
-        with:
-          node-version: 20
-
-      - name: Install dependencies
-        run: npm ci
-
-      - name: Test
+      - name: E2E Web - Credit only
         env:
           E2E_RTC_WEB_ROOT_URL: https://web.env.reactivetrader.com/pull/${{ github.event.number }}
-        run: npm run e2e:web
-
-      - name: Artifact Upload
-        uses: actions/upload-artifact@v4
-        if: failure()
-        with:
-          name: playwright-report
-          path: packages/client/playwright-report/
-          retention-days: 7
-
-  openfin-end-to-end-test-fx:
-    name: Openfin e2e test - FX
-
-    needs: web-end-to-end-test
-
-    runs-on: windows-latest
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Set Node version
-        uses: actions/setup-node@v4
-        with:
-          node-version: 20
-
-      - name: Install dependencies
-        run: npm ci
-
-      - name: start openfin
-        run: npm run _e2e:openfin:run --manifest_url=https://openfin.env.reactivetrader.com/pull/${{ github.event.number }}/config/rt-fx.json
-
-      - name: Run tests
-        env:
-          E2E_RTC_WEB_ROOT_URL: https://openfin.env.reactivetrader.com/pull/${{ github.event.number }}
-        run: npm run e2e:openfin fx-spot-tile.spec fx-blotter.spec fx-analytics.spec
-
-      - name: Artifact Upload
-        uses: actions/upload-artifact@v4
-        if: failure()
-        with:
-          name: playwright-report
-          path: packages/client/playwright-report/
-          retention-days: 7
-
-  openfin-end-to-end-test-credit:
-    name: Openfin e2e test - Credit
-
-    needs: web-end-to-end-test
-
-    runs-on: windows-latest
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Set Node version
-        uses: actions/setup-node@v4
-        with:
-          node-version: 20
-
-      - name: Install dependencies
-        run: npm ci
-
-      - name: start openfin
-        run: npm run _e2e:openfin:run --manifest_url=https://openfin.env.reactivetrader.com/pull/${{ github.event.number }}/config/rt-credit.json
-
-      - name: Run tests
-        env:
-          E2E_RTC_WEB_ROOT_URL: https://openfin.env.reactivetrader.com/pull/${{ github.event.number }}
-        run: npm run e2e:openfin credit.spec
+        run: npx playwright test --project=web credit.spec.ts
 
       - name: Artifact Upload
         uses: actions/upload-artifact@v4
-        if: failure()
+        # if: failure()
```

Comment:
Intentional?

Reply:
yes, although pointless, as the playwright config won't do screenshots / trace unless failure .. some of this is not really joined up .. but that's a background task, tightening up the E2E a bit

Reply:
no longer pointless, screenshots and trace now working and posted (briefly) against all E2E runs on this branch